# app name
string name
# user-friendly display name of application
string display_name
# icon for showing app
Icon icon
# ordered list (by preference) of client applications to interact with this robot app.  
ClientApp[] client_apps
